packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBar.tsx

```diff
@@ -17,14 +17,16 @@ import { css } from '@emotion/css';
 import { groupBy as _groupBy } from 'lodash';
 import React from 'react';
 import { compose, onlyUpdateForKeys, withProps, withState } from 'recompose';
-import { autoColor, createStyle, Theme } from '../Theme';
+import { GrafanaTheme2 } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
+import { autoColor } from '../Theme';
 import { TraceSpan } from '../types/trace';
 import { TNil } from '../types';
 import { UIPopover } from '../uiElementsContext';
 import AccordianLogs from './SpanDetail/AccordianLogs';
 import { ViewedBoundsFunctionType } from './utils';
 
-const getStyles = createStyle((theme: Theme) => {
+const getStyles = stylesFactory((theme: GrafanaTheme2) => {
```

Comment:
SpanBar is functional component so we can use `useStyles2` and remove theme passed from `SpanBarRow`?

Reply:
good point, changed

Reply:
LGTM now!